### PR TITLE
Support make as CMake Generator

### PR DESCRIPTION
**Pull request by eddyashton**
Trying to build with `make` rather than `ninja` would produce an error, saying that the enclave `.so` files weren't generated. This fixes that. `make` is _much_ slower than `ninja`, so I don't think we need to maintain this, but its nice if it works.

### CODE REVIEW

**Comment by achamayou** [comment]
If we're going to support it, let's add it to the Daily.

**Comment by achamayou** [comment]
```
mkdir make_build
cd make_build
cmake ..
```

File: .azure-pipelines-templates/daily-matrix.yml [hunk 1]
```diff
@@ -19,24 +19,37 @@ parameters:
...
+  #   parameters:
+  #     target: SGX
+  #     env: '${{ parameters.env.SGX }}'
+  #     cmake_args: '${{ parameters.build.install.cmake_args }} ${{ parameters.build.SGX.cmake_args }} -DTLS_TEST=ON -DSHUFFLE_SUITE=ON'
+  #     suffix: 'Release'
+  #     artifact_name: 'SGX_Release'
+  #     ctest_filter: '-LE "benchmark|perf"'
+  #     ctest_timeout: '240'
+  #     install_prefix: '${{ parameters.build.install.install_prefix }}'
+  #     tls_report: true
```
Comment (achamayou) [inline .azure-pipelines-templates/daily-matrix.yml:56]:
```suggestion
  - template: common.yml
    parameters:
      target: NoSGX
      env: '${{ parameters.env.NoSGX }}'
      cmake_args: '${{ parameters.build.debug.cmake_args }} ${{ parameters.build.NoSGX.cmake_args }}'
      suffix: 'Coverage'
      artifact_name: 'NoSGX_Coverage'
      ctest_filter: '-LE "benchmark|perf|long_test"'
      ctest_timeout: '300'

  - template: common.yml
    parameters:
      target: SGX
      env: '${{ parameters.env.SGX }}'
      cmake_args: '${{ parameters.build.install.cmake_args }} ${{ parameters.build.SGX.cmake_args }} -DTLS_TEST=ON -DSHUFFLE_SUITE=ON'
      suffix: 'Release'
      artifact_name: 'SGX_Release'
      ctest_filter: '-LE "benchmark|perf"'
      ctest_timeout: '240'
      install_prefix: '${{ parameters.build.install.install_prefix }}'
      tls_report: true
```

Comment (eddyashton) [reply .azure-pipelines-templates/daily-matrix.yml]: Done - just had these disabled temporarily to test the new job in isolation.

**Pull request merged**